source/ShopPanel.cpp

```diff
@@ -760,14 +760,23 @@ bool ShopPanel::Click(int x, int y, int /* clicks */)
 	if(button)
 		return DoKey(button);
 
-	// Check for clicks in the scroll arrows.
+	// Check for clicks in the ShipsSidebar Panel arrows.
 	if(x >= Screen::Right() - 20)
 	{
 		if(y < Screen::Top() + 20)
 			return Scroll(0, 4);
 		if(y < Screen::Bottom() - BUTTON_HEIGHT && y >= Screen::Bottom() - BUTTON_HEIGHT - 20)
 			return Scroll(0, -4);
 	}
+	// Check for clicks in the DetailsSidebar Panel arrows.
```

Comment:
```suggestion
	// Check for clicks on the DetailsSidebar pane arrows.
```
They aren't really their own panels.